db/rdbms/migration/0002_migrate_descriptor_to_extended_descriptor.go

```diff
@@ -60,19 +62,29 @@ type DescriptorMigration struct {
 	log *logrus.Entry
 }
 
+type dbConn interface {
+	Exec(query string, args ...interface{}) (sql.Result, error)
+	Query(query string, args ...interface{}) (*sql.Rows, error)
+}
+
+type jobDescriptorPairs struct {
+	jobIDs              []types.JobID
+	extendedDescriptors []string
+}
```

Comment:
Just a nit, why not make this
```
type jobDescriptorPair struct {
	jobID              types.JobID
	extendedDescriptor string
}
```
and then below having
```
var updates []jobDescriptorPair
```
so they are guaranteed to stay together without being super careful with the indices?

Reply:
Heh, I have gone through several iterations, and your proposal was definitely one of the approaches, in one of the intermediate states. I don't think there is any reason not to adopt that approach in this version, which I like much more.